products/jhipster.md

```diff
@@ -0,0 +1,70 @@
+---
+title: JHipster
+category: app
+versionCommand: yo jhipster
+permalink: /jhipster
+releasePolicyLink: https://www.jhipster.tech/releases/
+changelogTemplate: "https://github.com/jhipster/generator-jhipster/tag/__LATEST__"
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Supported
+#iconSlug: jhipster , WIP see https://github.com/jhipster/generator-jhipster/issues/20533
+
+#auto:
+#-   maven: org.apache.maven/maven-core
+releases:
+-   releaseCycle: "7"
+    eol: false
```

Comment:
`7` being the latest version I think it should also be marked as supported :

    support: true